components/brave_ads/core/browser/BUILD.gn

```diff
@@ -5,6 +5,8 @@
 
 static_library("browser") {
   sources = [
+    "internals/ads_internals_handler.cc",
```

Comment:
these should have their own target/BUILD.gn in `internals`

Reply:
Fixed